doc/holochain_101/src/first_steps.md

```diff
@@ -1,7 +1,7 @@
 # First steps writing Holochain hApps with Rust
 
 ___
-This tutorial builds for the 0.0.9-alpha release but as the API and HDK are changing it will likely fail under newer releases.
+This tutorial builds for the 0.0.18-alpha1 release but as the API and HDK are changing it will likely fail under newer releases.
```

Comment:
@zippy this one can't be updated unless the tutorial is actually updated too, which I think someone is doing that open source in another PR

Reply:
Agreed. What I'll do is I'll update to the latest release and check that the tutorial works fine.